packages/core/src/sdk/deliveryPromise/useDeliveryPromise.ts

```diff
@@ -298,74 +298,69 @@ export function useDeliveryPromise({
             key !== DELIVERY_OPTIONS_FACET_KEY &&
             key !== DYNAMIC_ESTIMATE_FACET_KEY
         )
-      : allFacets
-          .filter(({ key }) => key !== DYNAMIC_ESTIMATE_FACET_KEY) // TODO: remove this filter when dynamic estimate is implemented
-          .map((facet) => {
-            if (facet.__typename !== 'StoreFacetBoolean') return facet
-
-            if (facet.key === DELIVERY_OPTIONS_FACET_KEY) {
-              facet.values = withUniqueFacet(
-                facet.values,
-                allDeliveryOptionsFacet
-              )
-
-              return facet
-            }
-
-            if (facet.key !== SHIPPING_FACET_KEY) return facet
+      : allFacets.map((facet) => {
+          if (facet.__typename !== 'StoreFacetBoolean') return facet
```

Comment:
the only change I've made in this file was removing `.filter(({ key }) => key !== DYNAMIC_ESTIMATE_FACET_KEY)` so I can display the dynamic estimate facet. The other changes are indentation 🤔

Reply:
I made a few changes in this hook in the latest commits.